src/FirePokemon.java

```diff
@@ -0,0 +1,49 @@
+package src;
+
+public class FirePokemon extends Pokemon {
+
+    //VARIABLES
```

Comment:
Dit soort commentaar is nuttig voor jezelf, maar wel overbodig. Ik kan zonder het commentaar al zien dat hieer variabelen staan. Ik kan ook zonder het commentaar al zien waar de constructor staat (per conventie staat die onder de variabelen en dat heb jij ook). En ook de getters en setter behoeven geen commentaar. In de naam van de methode staat immers al "get..." of "set..."